framework_lib/src/commandline/mod.rs

```diff
@@ -513,11 +513,11 @@ pub fn run_with_args(args: &Cli, _allupdate: bool) -> i32 {
         if let Some((mic, cam)) = print_err(ec.get_privacy_info()) {
             println!(
                 "Microphone privacy switch: {}",
-                if mic { "Open" } else { "Closed" }
+                if mic { "Enabled" } else { "Disabled" }
             );
             println!(
                 "Camera privacy switch:     {}",
-                if cam { "Open" } else { "Closed" }
+                if cam { "Enabled" } else { "Disabled" }
             );
```

Comment:
Maybe we could add some extra context like:

```
Privacy Slider (Open/Black = Device enabled, Closed/Red = Device disabled)
  Microphone: Closed
  Camera:     Open
```

Reply:
A switch being closed means that it is connected and that current can flow. These are electrical switches (both physically, and in the mental model of what they do). I cannot "Closed" cannot mean "Disconnected". 

New proposal:
```
Privacy Slider (Connected/Black = Device Enabled; Disconnected/Red = Device Disabled)
  Microphone: Connected
  Camera:     Disconnected
```

I think that connected/disconnected best describes the state. I think that Open/Closed can describe it, if used correctly (following switch conventions). I think that Enabled/Disabled describes the function well, but loses the physical connection.

From [the Wikipedia article on switches](https://en.wikipedia.org/wiki/Switch):
> Each set of contacts can be in one of two states: either "closed" meaning the contacts are touching and electricity can flow between them, or "open", meaning the contacts are separated and the switch is nonconducting.

Reply:
I like "connected"/"disconnected" that should be good